unittests/ThunkLibs/generator.cpp

```diff
@@ -332,7 +332,10 @@ SourceWithAST Fixture::run_thunkgen_host(std::string_view prelude, std::string_v
         "};\n"
         "template<typename F>\n"
         "void FinalizeHostTrampolineForGuestFunction(F*);\n"
-        "struct ExportEntry { uint8_t* sha256; void(*fn)(void *); };\n";
+        "struct ExportEntry { uint8_t* sha256; void(*fn)(void *); };\n"
+        "inline void *dlsym_default(void *__restrict handle, const char *__restrict symbol) {\n"
```

Comment:
A declaration without definition should suffice here, since we don't involve this function body in AST testing.
```suggestion
        "void *dlsym_default(void* handle, const char* symbol);\n"
```

Reply:
Removed the body.